examples/docker/taxi-on-omnisci/nyc-taxi-omnisci.dockerfile

```diff
@@ -41,11 +41,14 @@ RUN wget -nv https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.
     rm -f /tmp/miniconda3.sh && \
     echo ". '${CONDA_DIR}/etc/profile.d/conda.sh'" >> "${HOME}/.profile"
 
+# modin==0.10.1 comes with an error: https://github.com/modin-project/modin/issues/3215
+# TODO: update modin version after a new version is released
 RUN conda update -n base -c defaults conda -y && \
     conda config --set channel_priority strict && \
     conda create -n modin --yes --no-default-packages && \
     conda activate modin && \
-    conda install -c intel/label/modin -c conda-forge modin "ray>=1.0.0" "numpy==1.20.3" && \
+    conda install -c intel/label/modintest -c conda-forge "omniscidbe4py>=v5.5.0" && \
+    pip install "modin[ray]==0.10.0" && \
```

Comment:
Isn't `0.10.1` the latest release of modin? Should we use it here? Is it available in `intel/label/modintest` conda channel?

Reply:
`0.10.1` version could not be used as it had a bug preventing `nyc-taxi` example from running.

On the other hand, the bug shouldn't be in `0.10.2` version, but we need to wait until it appears in `conda-forge` channel.

Reply:
0.10.2 is in conda-forge now.